examples/common_vmseries/README.md

```diff
@@ -194,7 +194,7 @@ terraform destroy
 | <a name="input_vmseries_username"></a> [vmseries\_username](#input\_vmseries\_username) | Initial administrative username to use for all systems. | `string` | `"panadmin"` | no |
 | <a name="input_vmseries_password"></a> [vmseries\_password](#input\_vmseries\_password) | Initial administrative password to use for all systems. Set to null for an auto-generated password. | `string` | `null` | no |
 | <a name="input_availability_set"></a> [availability\_set](#input\_availability\_set) | A map defining availability sets. Can be used to provide infrastructure high availability when zones cannot be used.<br><br>Following properties are supported:<br>- `name` - name of the Application Insights.<br>- `update_domain_count` - specifies the number of update domains that are used, defaults to 5 (Azure defaults).<br>- `fault_domain_count` - specifies the number of fault domains that are used, defaults to 3 (Azure defaults).<br><br>Please keep in mind that Azure defaults are not working for each region (especially the small ones, w/o any Availability Zones). Please verify how many update and fault domain are supported in a region before deploying this resource. | `any` | `{}` | no |
-| <a name="input_application_insights"></a> [application\_insights](#input\_application\_insights) | A map defining Azure Application Insights. There are three ways to use this variable:<br><br>* when the value is set to `null` (default) no AI is created<br>* when the value is a map containing `name` key (other keys are optional) a single AI instance will be created under the name that is the value of the `name` key<br>* when the value is an empty map or a map w/o the `name` key, an AI instance per each VMSeries VM will be created. All instances will share the same configuration. All instances will have names corresponding to their VM name.<br><br>Names for all AI instances are prefixed with `var.name_prefix`.<br><br>Properties supported (for details on each property see [modules documentation](../modules/application\_insights/README.md)):<br><br>- `name` : (optional, string) a name of a single AI instance<br>- `workspace_mode` : (optional, bool) defaults to `true`, use AI Workspace mode instead of the Classical (deprecated)<br>- `workspace_name` : (optional, string) defaults to AI name suffixed with `-wrkspc`, name of the Log Analytics Workspace created when AI is deployed in Workspace mode<br>- `workspace_sku` : (optional, string) defaults to PerGB2018, SKU used by WAL, see module documentation for details<br>- `metrics_retention_in_days` : (optional, number) defaults to current Azure default value, see module documentation for details<br><br>Example of an AIs created per VM, in Workspace mode, with metrics retention set to 1 year:<pre>vmseries = {<br>  'vm-1' = {<br>    ....<br>  }<br>  'vm-2' = {<br>    ....<br>  }<br>}<br><br>application_insights = {<br>  metrics_retention_in_days = 365<br>}</pre> | `map(string)` | `null` | no |
```

Comment:
@jamesholland-uk you need to change this in the `variables.tf` and then re-generate the README file.

Otherwise the release CI will fail as it tries to run `terraform_docs` and looks for changes:

![image](https://github.com/PaloAltoNetworks/terraform-azurerm-vmseries-modules/assets/42772730/80454764-7798-4b1b-80ef-ef41d670168e)